dev/tox.ini

```
@@ -14,7 +14,7 @@
 # limitations under the License.
 
 [pycodestyle]
-ignore=E226,E241,E305,E402,E722,E731,E741,W503,W504
+ignore=E226,E241,E305,E402,E722,E731,E741,W503,W504,E231,E203
```

```suggestion
ignore=E203,E226,E231,E241,E305,E402,E722,E731,E741,W503,W504
```
(to be consistent with how codes are ordered throughout the file)

---

Thanks, @deepyaman